nilearn/plotting/img_plotting.py

```diff
@@ -807,7 +807,7 @@ def _plot_roi_contours(display, roi_img, cmap, alpha, linewidths):
     roi_img = _utils.check_niimg_3d(roi_img)
     roi_data = get_data(roi_img)
     labels = np.unique(roi_data)
-    cmap = plt.get_cmap(cmap)
+    cmap = plt.cm.ColormapRegistry.get_cmap(cmap)
```

Comment:
FYI: this won't work as you need to instantiate `plt.cm.ColormapRegistry` first.

the matplotlib doc suggests that in general you should not need to do this.